packages/blockchain-wallet-v4/src/redux/kvStore/buySell/selectors.js

```diff
@@ -9,3 +9,10 @@ export const getSfoxTrades = state => getMetadata(state).map(path(['value', 'sfo
 export const getSfoxUser = state => getMetadata(state).map(path(['value', 'sfox', 'user']))
 
 export const getCoinifyTrades = state => getMetadata(state).map(path(['value', 'coinify', 'trades']))
+
+export const getBuySellTxHashMatch = (state, hash) => getMetadata(state).map(data => {
+  const allTrades = concat(getSfoxTrades(state).getOrElse([]), getCoinifyTrades(state).getOrElse([]))
+  const tradeHashes = allTrades.map(t => t.tx_hash)
+  const shouldHaveLabel = contains(hash, tradeHashes)
+  return shouldHaveLabel && 'buy-sell'
+}).getOrElse(false)
```

Comment:
I don't think we call getOrElse directly in a core selector function anywhere else